client/my-sites/email/email-providers-in-depth-comparison/in-depth-comparison/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { FunctionComponent } from 'react';
+import type { TranslateResult } from 'i18n-calypso';
+import type { ReactElement } from 'react';
+
+export type ProviderComparison = {
+	header: ReactElement;
+	tools: TranslateResult;
+	storage: TranslateResult;
+	importing: TranslateResult;
+	support: TranslateResult;
+	footerBadge?: ReactElement;
```

Comment:
We don't define that particular prop in the other component, let's remove it for now.